docs/filters.md

```diff
@@ -727,3 +727,28 @@ headerToQuery("X-Foo-Header", "foo-query-param")
 
 The above filter will set `foo-query-param` query param respectively to the `X-Foo-Header` header
 and will override the value if the queryparam exists already
+
+## queryToHeader
+
+Filter which assigns the value of a given query param from the
+incoming Request to a given Header with optional static prefix value.
+
+Parameters:
+
+* The name of the query param key to pick from request
+* The name of the header to add to request
+* The static prefix string to prepend to the header value
+
+Examples:
+
+```
+headerToQuery("foo-query-param", "X-Foo-Header")
+headerToQuery("access_token", "Authorization", "Bearer ")
```

Comment:
This should be `queryToHeader`

Reply:
I also think the prepending of the value is a bit magic, would it make sense to make it a format string? i.e.:

```
queryToHeader("access_token", "Authorizaton", "Bearer %s")
```